Source/Response.swift

```diff
@@ -77,4 +77,16 @@ public extension Response {
         }
         return string
     }
+
+    /// Maps received data at key path into a String.
+    public func mapString(atKeyPath keyPath: String?) throws -> String {
```

Comment:
Hmm, why not? I think using one function is more clear. This would also prevent possible code duplication.